CMakeLists.txt

```diff
@@ -125,7 +125,8 @@ endif()
 if (NOT DEFINED QL_BOOST_COMPONENTS)
     set(QL_BOOST_COMPONENTS "")
     # Boost thread needed for sessions, singleton thread-safe init and thread-safe observer pattern
-    if(QL_ENABLE_SESSIONS OR QL_ENABLE_SINGLETON_THREAD_SAFE_INIT OR QL_ENABLE_THREAD_SAFE_OBSERVER_PATTERN)
+    if(QL_ENABLE_PARALLEL_UNIT_TEST_RUNNER OR QL_ENABLE_SESSIONS 
```

Comment:
Hi Klaus, I removed the dependency on Boost thread in #1373 for the parallel unit test runner. Did you see any compiler or linker error after that was merged?

Reply:
Hi, I'm sorry, my bad. I've removed the change.